src/Database/ScopeVisibilityTrait.php

```diff
@@ -12,19 +12,51 @@
 use Flarum\Event\ScopeModelVisibility;
 use Flarum\User\User;
 use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Arr;
 
 trait ScopeVisibilityTrait
 {
+    protected static $visibilityScopers = [];
+    protected static $DEFAULT = '*';
```

Comment:
Because it's a constant, but we can't use constants in traits (thanks PHP!)

Reply:
:thinking: then I'd personally prefer if it were lowercase, because it's not technically a constant, we'd want it to be, but it's not xD

Reply:
To me, convention like naming is more about what something represents than what it technically is. Using uppercase naming indicates to readers that it should be considered as a constant.

Reply:
Since it's not actually a constant I'm going to vote for lower case. I understand that we want it to be a constant in the way it's used, but it isn't actually one. In theory we could make an interface with the constant and then implement that.... But that's just adds extra unneeded files.

Reply:
Done